source

```diff
@@ -88992,47 +88998,56 @@ dictionary <dfn>PromiseRejectionEventInit</dfn> : <span>EventInit</span> {
 
-     <li>
-      <p>If there are <span data-x="top-level browsing context">top-level browsing contexts</span>
-      <var>B</var> that the user agent believes would not benefit from having their rendering
-      updated at this time, then remove from <var>docs</var> all <code>Document</code> objects whose
-      <span data-x="concept-document-bc">browsing context</span>'s <span>top-level browsing
-      context</span> is in <var>B</var>.</p>
-
-      <div class="note">
-       <p>Whether a <span>top-level browsing context</span> would benefit from having its rendering
-       updated depends on various factors, such as the update frequency. For example, if the browser
-       is attempting to achieve a 60Hz refresh rate, then these steps are only necessary every 60th
-       of a second (about 16.7ms). If the browser finds that a <span>top-level browsing
-       context</span> is not able to sustain this rate, it might drop to a more sustainable 30Hz for
-       that set of <code>Document</code>s, rather than occasionally dropping frames. (This
-       specification does not mandate any particular model for when to update the rendering.)
-       Similarly, if a <span>top-level browsing context</span> is in the background, the user agent
-       might decide to drop that page to a much slower 4Hz, or even less.</p>
-
-       <p>Another example of why a browser might skip updating the rendering is to ensure certain
-       <span data-x="concept-task">tasks</span> are executed immediately after each other, with only
-       <span data-x="perform a microtask checkpoint">microtask checkpoints</span> interleaved (and
-       without, e.g., <span data-x="run the animation frame callbacks">animation frame
-       callbacks</span> interleaved). For example, a user agent might wish to coalesce timer
-       callbacks together, with no intermediate rendering updates.</p>
-      </div>
-     </li>
-
-     <li>
-      <p>If there are a <span data-x="nested browsing context">nested browsing contexts</span>
-      <var>B</var> that the user agent believes would not benefit from having their rendering
-      updated at this time, then remove from <var>docs</var> all <code>Document</code> objects whose
-      <span data-x="concept-document-bc">browsing context</span> is in <var>B</var>.</p>
-
-      <p class="note">As with <span data-x="top-level browsing context">top-level browsing
-      contexts</span>, a variety of factors can influence whether it is profitable for a browser to
-      update the rendering of <span data-x="nested browsing context">nested browsing
-      contexts</span>. For example, a user agent might wish to spend less resources rendering
-      third-party content, especially if it is not currently visible to the user or if resources are
-      constrained. In such cases, the browser could decide to update the rendering for such content
-      infrequently or never.</p>
-     </li>
+     <li id="step-7-3-rendering-opportunities">
```

Comment:
Don't bake step numbers into IDs or references; they'll change over time. You could instead use labels; search for "step labeled" in the spec to see some examples.

Reply:
Done.